src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.Scanner;
+import java.util.Collections;
+
+
 public class Duke {
     public static void main(String[] args) {
+        showLogo();
+        showWelcomeMsg();
+        readUserCmd();
+        showGoodbyeMsg();
+    }
+
+    private static void showLogo() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println(logo);
+    }
+
+    private static final String END_CMD = "bye";
+    private static final int MAX_TASK = 100;
+    private static final int SEPARATOR_LEN = 50;
+    private static int taskCount = 0;
+    private static Task[] tasks = new Task[MAX_TASK];
+
+    private static void showSeparator() {
+        String separator = String.join("", Collections.nCopies(SEPARATOR_LEN, "="));
+        System.out.println(separator);
+    }
+
+    private static void showWelcomeMsg() {
+        showSeparator();
+        System.out.println("Hello! I'm Duke ^_^");
+        System.out.println("What can I do for you?");
+        showSeparator();
+    }
+
+    private static void showGoodbyeMsg() {
+        System.out.println("Bye. Hope to see you again soon! qwq");
+        showSeparator();
+    }
+
+    private static void showTaskCount() {
+        System.out.printf("Now we have %d tasks in the list.\n", taskCount);
+        showSeparator();
+    }
+
+    private static void readUserCmd() {
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+        while (!input.equals(END_CMD)) {
+            String[] args = input.split(" ", 2);
+            String arg1 = args[0];
+            String arg2 = "";
+            if (args.length > 1) {
+                arg2 = args[1];
+            }
+            switch (arg1) {
```

Comment:
Great work on following coding standard for switch and case! It is a little tricky to watch out for!